### PR TITLE
feat(Task-20): As a Product Manager, I want to increase the potential users of my application

### PR DESCRIPTION
**Pull request by jLagosCarrera**

### CODE REVIEW

File: src/app/shared/filters/capitalize.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+export default (input, all) => {
+    const REGEX_ALL = /([^\W_]+[^\s-]*) */g;
+    const REGEX_FIRST = /([^\W_]+[^\s-]*)/;
+
+    if (!input) return '';
```
Comment (eloydotco) [inline src/app/shared/filters/capitalize.js]: Please, use the complete `if {} else {}` format

File: src/app/shared/filters/capitalize.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+    const regex = (all) ? REGEX_ALL : REGEX_FIRST;
+    return input.replace(regex, (txt) =>
+        txt.charAt(0).toUpperCase() + txt.substr(1).toLowerCase());
+
```
Comment (eloydotco) [inline src/app/shared/filters/capitalize.js]: and remove this empty line

File: src/app/shared/filters/capitalize.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+export default (input, all) => {
+    const REGEX_ALL = /([^\W_]+[^\s-]*) */g;
+    const REGEX_FIRST = /([^\W_]+[^\s-]*)/;
+
+    if (!input) return '';
+    const regex = (all) ? REGEX_ALL : REGEX_FIRST;
+    return input.replace(regex, (txt) =>
+        txt.charAt(0).toUpperCase() + txt.substr(1).toLowerCase());
```
Comment (eloydotco) [inline src/app/shared/filters/capitalize.js]: don't break arrow functions

File: src/resources/translations/es.js [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+        "INPUT_MESSAGE": "Mensaje",
+        "INPUT_NAME": "Tu nombre",
+        "TITLE": "Contáctanos!",
+        "TOOLTIPS": {
+            "EXIT": "Salir",
+            "INSERT_EMAIL": "Introduce tu email aquí",
+            "INSERT_ISSUE": "Introduce tu problema aquí",
+            "INSERT_MESSAGE": "Explícanos tu problema aquí",
+            "INSERT_NAME": "Introduce tu nombre aquí",
+            "SEND_ISSUE": "Envia el problema"
```
Comment (eloydotco) [inline src/resources/translations/es.js]: typo :)

File: src/resources/translations/gl.js [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+        "INPUT_MESSAGE": "Mensaxe",
+        "INPUT_NAME": "Teu nome",
+        "TITLE": "Contáctanos!",
+        "TOOLTIPS": {
+            "EXIT": "Saír",
+            "INSERT_EMAIL": "Introduce o teu email aquí",
+            "INSERT_ISSUE": "Introduce o teu problema aquí",
+            "INSERT_MESSAGE": "Explícanos o teu problema aquí",
+            "INSERT_NAME": "Introduce o teu nome aquí",
+            "SEND_ISSUE": "Envia o problema"
```
Comment (eloydotco) [inline src/resources/translations/gl.js]: typo :)

**Review by eloydotco — CHANGES_REQUESTED** [review]

**Review by eloydotco — APPROVED** [review]

File: src/app/shared/filters/capitalize.js [hunk 6]
```diff
@@ -0,0 +1,11 @@
...
+    const REGEX_ALL = /([^\W_]+[^\s-]*) */g;
+    const REGEX_FIRST = /([^\W_]+[^\s-]*)/;
+
+    if (!input) {
+        return '';
+    } else {
+        const regex = (all) ? REGEX_ALL : REGEX_FIRST;
+        return input.replace(regex, (txt) => txt.charAt(0).toUpperCase() + txt.substr(1).toLowerCase());
+    }
+}
```
Comment (gbtec-ext-jorgevillar) [inline src/app/shared/filters/capitalize.js:11]: I'm glad you use Filters, but I think you can do this with a CSS style way more easy:
`<selector> {
    text-transform: capitalize;
}`

File: src/app/home-page/homePage.component.html [hunk 7]
```diff
@@ -1,6 +1,7 @@
 <div class="flex-column justify-center align-center fill-content content-background">
     <web-logo class="flex-row justify-center" icon-size="100px"></web-logo>
     <city-search class="flex-row justify-center width-75 padding-y-20"></city-search>
-    <displayable-list class="width-75" display-icon="restore" list-to-display="$ctrl.latestSearches" list-type="Recent">
+    <displayable-list class="width-75" display-icon="restore" list-to-display="$ctrl.latestSearches"
+        list-type="{{ ::'DISPLAYABLE_LIST.TYPES.RECENT' | translate }}">
```
Comment (gbtec-ext-jorgevillar) [inline src/app/home-page/homePage.component.html]: What happens with the one time binding in the next story (about placing a switch laguage behavior)? Please, take care of this

File: src/app/search-page/searchPage.component.html [hunk 8]
```diff
@@ -11,22 +11,24 @@
         <div class="flex-row full-width justify-center">
             <div class="flex-column align-center full-width">
                 <displayable-list class="flex-column justify-center align-center full-width" use-header="true"
-                    display-icon="restore" list-to-display="$ctrl.latestSearches" list-type="Recent"></displayable-list>
+                    display-icon="restore" list-to-display="$ctrl.latestSearches"
+                    list-type="{{ ::'DISPLAYABLE_LIST.TYPES.RECENT' | translate }}"></displayable-list>
```
Comment (gbtec-ext-jorgevillar) [inline src/app/search-page/searchPage.component.html]: Same as above

File: src/app/search-page/searchPage.component.html [hunk 9]
```diff
@@ -11,22 +11,24 @@
...
             </div>
         </div>
         <div class="flex-row full-width justify-center">
             <div class="flex-column align-center full-width">
                 <displayable-list class="flex-column justify-center align-center full-width" use-header="true"
-                    display-icon="near_me" list-to-display="$ctrl.nearbyCities" list-type="Nearby"></displayable-list>
+                    display-icon="near_me" list-to-display="$ctrl.nearbyCities"
+                    list-type="{{ ::'DISPLAYABLE_LIST.TYPES.NEARBY' | translate }}"></displayable-list>
```
Comment (gbtec-ext-jorgevillar) [inline src/app/search-page/searchPage.component.html]: Same as above

File: src/resources/translations/es.js [hunk 10]
```diff
@@ -0,0 +1,82 @@
...
+    },
+    "DISPLAYABLE_LIST": {
+        "TOOLTIPS": {
+            "HEADER": "lista de ciudades {{ type }}s",
+            "NO_RESULTS": "No hay resultados",
+            "SINGLE_CARD": "Ciudad {{ type }}"
+        },
+        "TYPES": {
+            "NEARBY": "cercana",
+            "RECENT": "reciente"
```
Comment (gbtec-ext-jorgevillar) [inline src/resources/translations/es.js:33]: In English this to translations are capitalized

File: src/resources/translations/es.js [hunk 11]
```diff
@@ -0,0 +1,82 @@
...
+            "INSERT_EMAIL": "Introduce tu email aquí",
+            "INSERT_ISSUE": "Introduce tu problema aquí",
+            "INSERT_MESSAGE": "Explícanos tu problema aquí",
+            "INSERT_NAME": "Introduce tu nombre aquí",
+            "SEND_ISSUE": "Envía el problema"
+        }
+    },
+    "DISPLAYABLE_LIST": {
+        "TOOLTIPS": {
+            "HEADER": "lista de ciudades {{ type }}s",
```
Comment (gbtec-ext-jorgevillar) [inline src/resources/translations/es.js]: Shouldn,t this start with a capital letter?

File: src/resources/translations/es.js [hunk 12]
```diff
@@ -0,0 +1,82 @@
...
+        "TOOLTIPS": {
+            "BAR_INFO": "Porcentaje de {{ type }}"
+        },
+        "TYPES": {
+            "CLOUDS": "nubes",
+            "HUMIDITY": "humedad"
+        }
+    },
+    "SEARCH_PAGE": {
+        "CONTACT_BUTTON": "Necesitas ayuda?",
```
Comment (gbtec-ext-jorgevillar) [inline src/resources/translations/es.js]: missing '¿' at the beginning

File: src/resources/translations/gl.js [hunk 13]
```diff
@@ -0,0 +1,82 @@
...
+        "TOOLTIPS": {
+            "OPEN_CONTACT": "Abrir formulario de contacto"
+        }
+    },
+    "WEBLOGO": {
+        "TOOLTIPS": {
+            "INFO": "Volver á páxina de inicio"
+        }
+    }
+}
```
Comment (gbtec-ext-jorgevillar) [inline src/resources/translations/gl.js:82]: Same requests as the espanish translation

**Review by gbtec-ext-jorgevillar — CHANGES_REQUESTED** [review]

File: src/app/shared/filters/capitalize.js (hunk 6 above)
Comment (jLagosCarrera) [reply src/app/shared/filters/capitalize.js:11]: We already talked about it, new issue for this 👌

File: src/resources/translations/es.js (hunk 10 above)
Comment (jLagosCarrera) [reply src/resources/translations/es.js:33]: Different use on English and Spanish, that's why it isn't capitalized

File: src/resources/translations/es.js [hunk 14]
```diff
@@ -24,19 +24,19 @@ export default {
...
         "TOOLTIPS": {
-            "HEADER": "lista de ciudades {{ type }}s",
+            "HEADER": "Lista de ciudades {{ type }}s",
             "NO_RESULTS": "No hay resultados",
             "SINGLE_CARD": "Ciudad {{ type }}"
         },
         "TYPES": {
             "NEARBY": "cercana",
-            "RECENT": "reciente"
+            "RECENT": "recente"
```
Comment (gbtec-ext-jorgevillar) [inline src/resources/translations/es.js]: 'Reciente' was right

**Review by gbtec-ext-jorgevillar — APPROVED** [review]

**Pull request merged**